src/main/java/ch/njol/skript/lang/SkriptParser.java

```diff
@@ -655,6 +675,29 @@ private static String supportedEventsNames(Class<? extends Event>[] supportedEve
 		}
 	}
 
+	/**
+	 * Get an {@link UnparsedLiteral} using {@link #expr} if {@code allowUnparsedLiteral} is {@code true}
+	 * and {@link Classes#parseSimple(String, Class, ParseContext)} is not {@code null}.
```

Comment:
I don't really like this kind of description since the code already says the exact same thing. Explaining it in terms of behaviors is better: `If allowed, returns an UnparsedLiteral of the current expr and error. If unparsed literals are not allowed or there's no possible matching literal, prints errors and returns null.` for example.